pyproject.toml

```diff
@@ -4,11 +4,20 @@ version = "1.0.0-dev"
 dependencies = [
     "django~=5.0",
     "django-environ~=0.11",
-    "plastron-utils~=4.0",
+    # By default, Plastron Utils is installed from Github.
+    # To use a local version for development, uncomment the following 
+    # line and comment the plastron-utils line with the Github URL
+    # "plastron-utils~=4.0",
+    "plastron-utils@git+https://github.com/umd-lib/plastron.git@4.0.0#subdirectory=plastron-utils",
     "rdflib~=7.0",
+    "waitress",
```

Comment:
Please update the waitress and whitenoise dependencies to include a compatible version specifier (`~=`).